GitUI/Hotkey/HotkeySettingsManager.cs

```diff
@@ -289,6 +297,13 @@ public static HotkeySettings[] CreateDefaultSettings()
                     Hk(FormBrowse.Command.StashPop, Keys.Control | Keys.Alt | Keys.Down),
                     Hk(FormBrowse.Command.ToggleBetweenArtificialAndHeadCommits, Keys.Control | Keys.OemBackslash),
                     Hk(FormBrowse.Command.ToggleBranchTreePanel, Keys.Control | Keys.Alt | Keys.C)),
+                new HotkeySettings(
+                    RepoObjectsTree.HotkeySettingsName,
+                    Hk(RepoObjectsTree.Command.Delete, Keys.Delete),
+                    Hk(RepoObjectsTree.Command.Rename, Keys.F2),
+                    Hk(RepoObjectsTree.Command.Search, Keys.F3),
+                    Hk(RepoObjectsTree.Command.MultiSelect, Keys.Control | Keys.Space),
+                    Hk(RepoObjectsTree.Command.MultiSelectWithChildren, Keys.Control | Keys.Alt | Keys.Space)),
```

Comment:
Ctrl-Shift instead?
(Is this OK with Alt-Click to select remote/local tracked?)

Reply:
> Ctrl-Shift instead?

Valid point! So, #9723 should switch to `CTRL+SHIFT+Click`, too, @h0lg.
(It is even more related to selecting than the combination with `ALT`. I do not see a conflict with `SHIFT+Click`.)

Reply:
Changed and updated the PR description.